docs/topics/mta-rn-new-features-7-0-0.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: CONCEPT
+[id="rn-new-features-7-0-0_{context}"]
+= New features
+
+
+This section describes the new features of the Migration Toolkit for Applications (MTA) 7.0.0.
+
+.Red Hat Single Sign On (RHSSO) configuration
+
+MTA supports Red Hat Single Sign-On (RHSSO) configuration, so that users' roles can be added using `keycloak`to control Konveyor and MTA authorization mechanisms. See link:https://access.redhat.com/documentation/en-us/migration_toolkit_for_applications/6.0/html/user_interface_guide/mta-6-installing-web-console-on-openshift_user-interface-guide#red_hat_single_sign_on[Red Hat Single Sign-On]. 
```

Comment:
```suggestion
MTA supports Red Hat Single Sign-On (RHSSO) configuration, so that users' roles can be added using `keycloak`to control MTA authorization mechanisms. See link:https://access.redhat.com/documentation/en-us/migration_toolkit_for_applications/6.0/html/user_interface_guide/mta-6-installing-web-console-on-openshift_user-interface-guide#red_hat_single_sign_on[Red Hat Single Sign-On]. 
```